MAUI/ImageEditor/shape.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Shape Annotation in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about the shape annotation feature of Syncfusion .NET MAUI ImageEditor(SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Shape Annotation in .NET MAUI ImageEditor (SfImageEditor)
+
+The image editor control allows you to add various shapes with customizable settings
+
+## Shape types
+
+* Circle
+* Rectangle
+* Arrow
+* Line
+* Dotted
+* DoubleArrow
+* DottedArrow
+* DottedDoubleArrow
+
+## Customize shape settings
+
+The shape annotations can be customized using common annotation customization properties.
```

Comment:
You can customize the appearance of each shape using the `ShapeSettings`

Reply:
Changed a content